### PR TITLE
3 build user logging function

### LINKED ISSUE #3: Build User Logging Function
- [x] 1. build both buyer/seller login function
- [x] 2. store user information in Seller/Buyer Collections

### PR DESCRIPTION
**Pull request by burtlau**

### CODE REVIEW

**Comment by burtlau** [comment]
@ChunangXu

File: src/main/java/com/example/Homes/Config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    @Autowired
+    private UserService userService;
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+
+    @Bean
+    public SecurityFilterChain defaultFilterChain(HttpSecurity httpSecurity) throws Exception {
+        return httpSecurity
+                .csrf(csrf-> csrf.disable())
```
Comment (ChunangXu) [inline src/main/java/com/example/Homes/Config/SecurityConfig.java:29]: why disable csrf here?

Comment (ChunangXu) [inline src/main/java/com/example/Homes/Config/SecurityConfig.java:29]: why disable csrf here?

File: src/main/java/com/example/Homes/Web/UserDto.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    private String lastName;
+    private String email;
+    private String password;
+    public UserDto()
+    {
+
+    }
+
+    public UserDto(String userName, String firstName, String lastName, String email, String password) {
+        super();
```
Comment (ChunangXu) [inline src/main/java/com/example/Homes/Web/UserDto.java:15]: Is it necessary to call super class's constructor here? (it is not extending another class)

File: src/main/java/com/example/Homes/entity/User.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Getter
```
Comment (ChunangXu) [inline src/main/java/com/example/Homes/entity/User.java:11]: If we already have @Data, is it still necessary to use @Getter here?

**Review by ChunangXu — APPROVED** [review]

**Pull request merged**